html-forms/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <p>
+        <label for="email">Email</label>
+        <br />
+        <input type="text" name="email" id="email">
+        <br />
+        <label for="reason">Why is coding cool?</label>
+        <br />
+        <select name="reason" id="reason">
+          <option value="Select an option">Select an option</option>
+          <option value="Because it's interesting">Because it's interesting</option>
+          <option value="Because it's fun">Because it's fun</option>
+          <option value="Because it's powerful">Because it's powerful</option>
+        </select>
+      </p>
+      <label for="language">Favorite Language (choose one)</label>
```

Comment:
There's no `input` with an `id` that matches the `for` attribute, so this isn't doing anything. I would just turn this into a `p` element